.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "third-party/etl"]
+[submodule "firmware/third-party/etl"]
 	path = firmware/third-party/etl
-	url = git@github.com:ETLCPP/etl.git
+	url = https://github.com/ETLCPP/etl.git
```

Comment:
https cloning is more standard than ssh

Reply:
Never realized git modules differentiated between the two. With the introduction of PAT with GitHub https repo URL's, I believe ssh has become the standard. Nonetheless I don't think this matters much since we are not trying to push changes via the submodule, we just want to clone

Reply:
I had authentication issues when trying to initialize the submodule. I probably could have solve it with more work but the issue is avoided with https